examples/simultaneous_translation/docs/evaluation.md

```diff
@@ -5,22 +5,50 @@ The server provides information needed by the client and evaluates latency and q
 We use the Fairseq toolkit as an example but the evaluation process can be applied in an arbitary framework.
 
 ## Server
-The server code is provided and can be set up locally for development purposes. For example, to evaluate a text simultaneous test set,
+The server code is provided and can be set up locally for development purposes. 
+Server sends source words to client, and record the delay when receving the prediction.
+Here is an instruction on how to setup server for development.
+
+To run start a text server listening at port 12321, with `SRC_FILE` and `TGT_FILE` as raw source and target text.
+
+```shell
+python $fairseq/example/simultaneous_translation/eval/server.py \
+    --tokenizer 13a \
+    --src-file SRC_FILE \
+    --tgt-file TGT_FILE \
+    --scorer-type text \
+    --output $result_dir \
+    --port 12321
+```
+
+As to speech models, if you have gone through the Data Preparation in [baseline experiment](baseline.md), you can use the json file in $DATA_ROOT/data-bin/mustc_en_de, for example, dev.json. So we can start the server
 
 ```shell
-python $user_dir/eval/server.py \
+python $fairseq/example/simultaneous_translation/eval/server.py \
     --tokenizer 13a \
-    --src-file $src \
-    --tgt-file $tgt \
-    --scorer-type {text, speech} \
-    --output $result_dir/eval \
+    --tgt-file $DATA_ROOT/data-bin/mustc_en_de/dev.json \
+    --tgt-file-type json \
```

Comment:
This is a new option? Did you make it backward compatible with the default being the json?

Reply:
Yes the default now is json to be compatible with old code base